python/cudf_polars/cudf_polars/dsl/expressions/rolling.py

```diff
@@ -152,12 +156,166 @@ def do_evaluate(  # noqa: D102
 
 
 class GroupedRollingWindow(Expr):
-    __slots__ = ("options",)
-    _non_child = ("dtype", "options")
+    """
+    Compute a window ``.over(...)`` aggregation and broadcast to rows.
+
+    Notes
+    -----
+    - This expression node currently implements **grouped window mapping**
+      (aggregate once per group, then broadcast back), not rolling windows.
+    - It can be extended later to support `rolling(...).over(...)`
```

Comment:
question: when Polars supports this, would we be able to use `plc.rolling.rolling_window` instead where we could use `plc.rolling.make_range_windows` to make group-informed window bounds?